package.json

```diff
@@ -23,7 +23,8 @@
     "push_l10n": "node ./build/commands/scripts/commands.js push_l10n",
     "pull_l10n": "node ./build/commands/scripts/commands.js pull_l10n",
     "chromium_rebase_l10n": "node ./build/commands/scripts/commands.js chromium_rebase_l10n",
-    "lint": "node ./build/commands/scripts/commands.js lint",
+    "lint": "npm run format && node ./build/commands/scripts/commands.js lint",
```

Comment:
not sure if we actually want to add this to lint since people generally don't expect files to be changed when running lint.

Reply:
updated this so it aborts if git diff is non-empty; in case there's something wrong with the auto-formatted results, this allows them to be easily reverted

Reply:
in theory we can feed clang-format with `--dry-run` to avoid modifications. Also ideally we can specify exact lines to check, if we are able to extract them from git.

Reply:
Out of scope of this PR, of course, but linter should basically be a precommit hook

Reply:
agree that we shouldn't make lint modify anything, just fail if it is not right.